chrome/navigation.json

```diff
@@ -0,0 +1,736 @@
+            "moduleScope": "notifications",
+            "module": "./RootApp",
+            "pathname": "/settings/notifications",
+            "routes": [
+                {
+                    "id": "notifications-rhel",
+                    "title": "Red Hat Enterprise Linux",
+                    "pathname": [
+                        "/settings/notifications",
+                        "/settings/notifications/rhel"
+                    ]
+                },
+                {
+                    "id": "notifications-openshift",
+                    "title": "OpenShift",
+                    "pathname": "/settings/notifications/openshift"
+                },
+                {
+                    "id": "notifications-application-services",
+                    "title": "Application Services",
+                    "pathname": "/settings/notifications/application-services"
+                }
+            ]
+        },
+        {
+            "id": "connector",
+            "title": "Red Hat Connector",
+            "moduleScope": "connector",
+            "module": "./RootApp",
+            "pathname": "/settings/connector"
+        },
+        {
+            "id": "settings-applications",
+            "title": "Applications",
+            "permissions": [
+                {
+                    "method": "isOrgAdmin",
+                    "apps": [
+                        "cost-management"
+                    ]
+                }
+            ],
+            "pathname": "/settings/applications",
+            "moduleScope": "applications",
+            "module": "./RootApp",
+            "routes": [
+                {
+                    "id": "settings-applications-advisor",
+                    "title": "Advisor",
+                    "pathname": [
+                        "/settings/applications",
+                        "/settings/applications/advisor"
+                    ],
+                    "permissions": {
+                        "method": "isEntitled",
+                        "args": [
+                            "insights"
+                        ]
+                    }
+                },
+                {
+                    "id": "settings-applications-cost-management",
+                    "title": "Cost Management",
+                    "pathname": "/settings/applications/cost-management",
+                    "permissions": {
+                        "method": "isEntitled",
+                        "args": [
+                            "cost_management"
+                        ]
+                    }
+                }
+            ]
+        }
+    ],
+    "leftNav": [
+        {
+            "id": "application-services",
+            "title": "Application services",
+            "pathname": "/application-services",
+            "navItems": []
+        },
+        {
+            "id": "openshift",
+            "title": "Openshift",
+            "pathname": "/openshift",
+            "navItems": [
```

Comment:
After some discussions, we won't be doing this. It will be again hard to construct the actual tree of the nav. We will lift the module definitions and routes definition from the navigation schema.

The navigation schema will be purely a mirror of the DOM and after splitting the schema into multiple files (based on namespace) there is no benefit for doing this.